app/components/EulaModal.js

```diff
@@ -26,6 +26,15 @@ export const EulaModal = ({ selectedLocale, continueFunction }) => {
   // Pull the EULA in the correct language, with en as fallback
   const eulaPath = EULA_FILES[selectedLocale] || en;
 
+  // Any links inside the EULA should launch a separate browser otherwise you can get stuck inside the app
+  shouldStartLoadWithRequestHandler = (webViewState) => {
+    if (webViewState.url != 'about:blank') {
+      Linking.openURL(webViewState.url);
+      return false;
```

Comment:
Same thing with the boolean values here - kind of hard to understand without looking at the documentation. What do you think about something like this?

```js
const shouldStartLoadWithRequestHandler = webViewState => {
    let shouldLoadRequest = true;

    if (webViewState.url != 'about:blank') {
      Linking.openURL(webViewState.url);

      // Opening the request in the users default browser, so cancel the WebView load
      shouldLoadRequest = false;
    }

    return shouldLoadRequest;
  };
```